composer.json

```diff
@@ -16,6 +16,7 @@
 	"require-dev": {
 		"phpunit/phpunit": "~9.4",
 		"wmde/fundraising-phpcs": "~5.0",
+		"wmde/inspector-generator": "dev-main",
```

Comment:
If there are no issues left, I'd like to create a release of inspector-generator so it doesn't look so "dev-y".

Reply:
Yep, it's working okay. The code format doesn't match ours so I excluded the generated classes from CS/Stan.